pkgs/top-level/release-python.nix

```diff
@@ -16,6 +16,7 @@
       # Keep evaluating home-assistant, which is transitively affected
       # by home-assistant-chip-core consuming OpenSSL 1.1. Affects roughly
       # 800 jobs.
+      # https://github.com/home-assistant-libs/chip-wheels/issues/24
```

Comment:
I think we should not duplicate the link here, as we already mention the package and naturally thats the next place people will look.